dbbackup/management/commands/dbbackup.py

```diff
@@ -43,16 +45,29 @@ def handle(self, **options):
         self.compress = options.get('compress')
         self.encrypt = options.get('encrypt')
         self.storage = BaseStorage.storage_factory()
-        database_keys = (self.database,) if self.database else dbbackup_settings.DATABASES
-        for database_key in database_keys:
-            database = settings.DATABASES[database_key]
-            self.dbcommands = DBCommands(database)
+        if options.get('mongo'):
+            # Mongo specific handling.
+            # TODO: Add support for multiple mongo db defined in settings.
```

Comment:
But this dedicated backup setting, the specification of it should come from dbbackup, right? But how do we specify it? And then you connect to it using another setting, because any kind of nosql framework will use any kind of setting...

I think we should limit django-dbbackup to be "backup any db that has a django backend and is listed in settings.DATABASES".... seems reasonable IMO :)

But it's still doable, just put the mongo engine, and it runs :) Perhaps you can even use this engine in your project :)